packages/management/addons-manager/bundle/config/upstream/addons-manager.yaml

```diff
@@ -261,10 +261,10 @@ spec:
             cpu: 100m
             memory: 40Mi
         ports:
-          - containerPort: 9865
+          - containerPort: #@ data.values.tanzuAddonsManager.webhookServerPort
             name: webhook-server
             protocol: TCP
-          - containerPort: 18316
+          - containerPort: #@ data.values.tanzuAddonsManager.healthzPort
```

Comment:
Need to update deployment with this datavalue for healthz port.

Reply:
updated the healthz port with data value